flowmachine/flowmachine/features/subscriber/interevent_period.py

```diff
@@ -0,0 +1,146 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Intervent period statistics, such as the average and standard deviation of the
+duration between calls.
+"""
+import warnings
+from typing import List
+
+from ...utils.utils import parse_tables_ensuring_columns
```

Comment:
This doesn't seem to exist?

You should get an error bubbled up from `Table` if one of the events tables doesn't have the columns you're expecting.

Reply:
I think this was renamed to (or merged with) `verify_columns_exist_in_all_tables` in a previous PR.

Reply:
Just FIY, in a previous PR we talked about the benefits of being explicit about the nonexistence of columns rather than relying on the SQL command failing. 

That is the reason why we are using `verify_columns_exist_in_all_tables`.

I have corrected the wrong function call as well.

Reply:
Right, but it won't _be_ an SQL fail. You effectively ask for a set of tables, and each table must have columns thus, and `Table` will throw you an exception if one of the columns doesn't exist in the table - https://github.com/Flowminder/FlowKit/blob/a5ee3b15d554c97f5f089a542b65e429c44d7289/flowmachine/flowmachine/core/table.py#L108

Appreciate that this func isn't something introduced in this PR, but I don't get the duplication of functionality.

Reply:
@greenape What exactly is your suggestion to do here?

Reply:
I'm happy to ditch the extra function as I agree it is not a good idea to duplicate the functionality. But it is easier to do in another PR since other classes are already using this extra functionality.